netlify.toml

```diff
@@ -8,8 +8,7 @@
   ignore = "git diff --quiet HEAD^ HEAD -- . ':!docs'"
 
 [build.environment]
-  YARN_FLAGS = "--frozen-lockfile"
-  YARN_VERSION = "1.22.19"
```

Comment:
Deprecated flag https://docs.netlify.com/configure-builds/manage-dependencies/#yarn. No longer recommended to use. Netlify gets the version from the new, `.yarnrc.yml` file.